src/arpeng/app/templates/CreeDictionary/components/definition__elaboration.html

```diff
@@ -0,0 +1,5 @@
+<div class="definition__elaboration" data-cy="elaboration">
+{{ lemma.linguist_info.pos }}
+  • {{ lemma.linguist_info.morphologies|join:" " }}
```

Comment:
Are these hard-coded mid dots? Why not an unordered list?

Reply:
That is a great point.

Reply:
I tried that and the CSS was getting very fiddly so I will have to come back to this.